src/ensembl/src/content/app/browser/drawer/Drawer.scss

```diff
@@ -14,30 +16,38 @@
     text-align: center;
   }
 
-  dd {
-    width: 600px;
+  p {
+    margin: 0px;
   }
 
-  label {
-    float: left;
-    font-weight: $light;
-    text-align: right;
-    width: 150px;
+  .nextLine {
+    display: block;
+    margin-bottom: 10px;
   }
 
-  p {
-    margin: 0 0 20px 0;
+  .clearFix {
+    display: grid;
+    grid-template-columns: 100px 1fr;
+    grid-column-gap: 1em;
+    grid-row-gap: 1em;
+  }
+
+  .label {
+    font-weight: $light;
+    text-align: right;
+    width: 90px;
```

Comment:
This width, that leaves this empty right corner in the left column of the grid, is confusing and likely unnecessary:
 
![Untitled Diagram (1)](https://user-images.githubusercontent.com/6834224/72360413-d4937400-36e7-11ea-96d7-597c830f8602.png)

If its purpose is to add white space between the label and the details field, then let's do so by increasing the gap between the columns:

```
grid-column-gap: calc(1em + 10px);
```
or something to this effect

Reply:
The width again was from previous commits when using float but i also use the width to move the label to the left. Having removed it, it looks ok. we may need to get Andrea to review it and see what she says.